spring-cloud-dataflow-server-core/src/main/resources/schemas/mysql/jpa.sql

```diff
@@ -13,16 +13,21 @@
 
     create table AUDIT_RECORDS (
        id bigint not null,
-        audit_Action bigint,
+        audit_Action bigint NOT NULL,
```

Comment:
Not related to the scope of this PR but just curious to know if there is an implicit meaning of using uppercase letter at the second word on some of these column names (ex: audit_Action, audit_Operation etc.,)?

Reply:
It is a bit odd...